noise_model_name: GaussianNoiseModel
std_deviations:
    position: [0.8, 0.8, 0.8]
    orientation: [0.1, 0.1, 0.1]
type_noise:
    allowed_semantic_tags:
        Buildings
        Fences
        Ground
        GuardRail
        Pedestrians
        Poles
        RoadLines
        Roads
        Sidewalks
        Sky
        Terrain
        TrafficLight
        TrafficSigns
        Vegetation
        Vehicles
        Walls
        Water
